scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/engine/daemon/DaemonEnvironment.java

```diff
@@ -48,6 +54,30 @@ public StartTask startTaskFor(ContainerInstance i) {
         .build();
   }
 
+  public List<StopTask> stopTaskFor(final List<Task> tasks) {
+    final List<String> tasksToStop =
```

Comment:
This method (`stopTaskFor`) is named just like `startTaskFor`, but it's doing more than just mapping tasks to `StopTask` actions as `startTaskFor` does for `StartTask` actions.

Maybe consider splitting out the filtering to a separate method, and have this method only do the mapping to `StopTask` objects.

Reply:
Sure, will revise all the naming and restrucutre these actions.